samples/bluetooth/tmap_bmr/src/vcp_vol_renderer.c

```diff
@@ -0,0 +1,68 @@
+/** @file
+ *  @brief Bluetooth Volume Control Profile (VCP) Volume Renderer role.
+ *
+ *  Copyright (c) 2020 Bose Corporation
+ *  Copyright (c) 2020-2022 Nordic Semiconductor ASA
+ *  Copyright (c) 2022 Codecoup
+ *  Copyright 2023 NXP
```

Comment:
Do you know why NXP does not have the `(c)` on their copyrights?

Reply:
Company policy. :)

They say (c) means the same as "Copyright".